cpp/src/parquet/properties.h

```diff
@@ -452,19 +452,39 @@ class PARQUET_EXPORT WriterProperties {
       return this->disable_statistics(path->ToDotString());
     }
 
-    /// Enable integer type to annotate decimal type as below:
-    ///   int32: 1 <= precision <= 9
-    ///   int64: 10 <= precision <= 18
-    /// Default disabled.
-    Builder* enable_integer_annotate_decimal() {
-      integer_annotate_decimal_ = true;
+    /// Enable decimal logical type with 1 <= precision <= 18 to be stored as
+    /// integer physical type.
+    ///
+    /// According to the specs, DECIMAL can be used to annotate the following types:
+    /// - int32: for 1 <= precision <= 9.
+    /// - int64: for 1 <= precision <= 18; precision < 10 will produce a warning.
```

Comment:
"precision < 10 will produce a warning." is this true of our implementation? Maybe we should skip saying that.

Reply:
This is copied from the specs. In our impl no warning is produced because we have fixed physical types for decimals and guarantee that precision < 10 will not use int64 type.

Reply:
Then perhaps we should change to this? So that we document our libraries behavior, and not just what the spec says?

```suggestion
    /// - int64: for 10 <= precision <= 18.
```

Reply:
Once this is done, I think this is ready to be merged :)

Reply:
Fixed. Please check again.